src/main/java/com/unbxd/SchemaManager/Controller/SchemaController.java

```diff
@@ -0,0 +1,108 @@
+package com.unbxd.SchemaManager.Controller;
```

Comment:
why capital letter in package name, also schema manager doesn't make sense as a package name